source/_integrations/smarty.markdown

```diff
@@ -35,23 +35,4 @@ The integration has a fan platform to view and control the ventilation speed, an
 - Warning
 - Filter Change Timer
 
-## Configuration
```

Comment:
could you add ha_config_flow: true to the header?

It's too far out for me to suggest